pandas/tests/util/test_validate_fillna_kwargs.py

```diff
@@ -0,0 +1,46 @@
+import pytest
+
+from pandas.util._validators import validate_fillna_kwargs
+
+
+def test_no_value_or_method():
+
+    msg = "Must specify a fill 'value' or 'method'."
+
+    with pytest.raises(ValueError, match=msg):
+        validate_fillna_kwargs(None, None, None)
+
+
+@pytest.mark.parametrize("value",
```

Comment:
Similar comment on parametrization - I think it's great that you are thinking about that! But generally these combinations of parameters don't really differ conceptually from one another and keep testing the same thing, so OK to remove

Reply:
Agree, shall remove this parametrization.